tests/integration/ceph_base_file_test.go

```diff
@@ -575,9 +575,13 @@ func waitForFilesystemActive(k8sh *utils.K8sHelper, clusterInfo *client.ClusterI
 
 	logger.Infof("waiting for filesystem %q to be active", filesystemName)
 	for i := 0; i < utils.RetryLoop; i++ {
-		// start the rgw admin command
+		// run the ceph fs status command
 		stat, err := k8sh.MakeContext().Executor.ExecuteCommandWithCombinedOutput(command, args...)
 		if err != nil {
+			if strings.Contains(err.Error(), "exit status 1") {
```

Comment:
Is this fixing something? Why not waiting for the retry loop to be done?

Reply:
oh I was experimenting with this, i'll investigate this a bit more...

Reply:
It wasn't needed anymore, removed it